packages/dom/src/dom/document-has-uncollapsed-selection.js

```diff
@@ -16,6 +16,9 @@ import inputFieldHasUncollapsedSelection from './input-field-has-uncollapsed-sel
 export default function documentHasUncollapsedSelection( doc ) {
 	return (
 		documentHasTextSelection( doc ) ||
-		inputFieldHasUncollapsedSelection( doc.activeElement )
+		( !! doc.activeElement &&
+			inputFieldHasUncollapsedSelection(
+				/** @type {HTMLElement} */ ( doc.activeElement )
```

Comment:
Could you instead extend the JSDoc of the function? Mostly curious 😄

Reply:
Actually yes, I'll make this change 🙂